README.md

```diff
@@ -0,0 +1,76 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse! project.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
```

Comment:
- [OPTIONAL] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser. You can put the steps to clone and run the project in a new Getting Started section under the project's description.